.github/workflows/test.yml

```diff
@@ -31,27 +31,27 @@ jobs:
         shell: bash -l {0}
         run: |
           svgdigitizer digitize test/data/xy.svg
-          diff test/data/xy.csv test/data/xy.csv.expected
+          python -c 'import pandas; pandas.testing.assert_frame_equal(pandas.read_csv("test/data/xy.csv"), pandas.read_csv("test/data/xy.csv.expected"))'
```

Comment:
The old `diff` approach does not work anymore since the exact numeric output now apparently depends on the numpy version. Anyway, the differences are only in some digits that certainly have no meaning anymore as you can see in the changes to the CSV files below.
I had to update the CSVs because the first line was always duplicated and the last line was missing, see #50. I also updated the rest to the output I now get with Python 3.9. I checked that nothing substantial has changed there except for a few trailing digits.